crates/jet1090/src/main.rs

```diff
@@ -70,6 +71,9 @@ struct Options {
     // - `port` must be a number
     // - `reference` can be LFPG for major airports, `43.3,1.35` otherwise
     sources: Vec<cli::Source>,
+
+    #[arg(short, long, value_name = "FILE", default_value = "jet1090.log")]
+    log_file: std::path::PathBuf,
```

Comment:
I would rather have this an `Option<std::path::PathBuf>` and only add it to the registry if it is not None. Otherwise, we cannot disable the logging.

Is there a way to have some rolling on the files in case the process runs for a long time?

Reply:
Also the tests fail because there is no log_file entry in the sample config file, but it will be fixed by itself once we switch to `Option<>`

Reply:
It makes sense, changes has been made.